src/utils/fhir.js

```diff
@@ -37,21 +38,30 @@ function generateOperationOutcome(code, msg) {
 }
 
 /**
- * Helper method to GET resources (and all included references) from a FHIR server
+ * Helper method to GET resources (and all included references) from a FHIR server and
+ * save them to the database.
  *
  * @param {string} server - the sourse server base url
- * @param {string} type - the type of the resource
- * @param {string[]} ids - (optional) list of ids to get
- * @returns promise of axios data
+ * @param {string} resourceType - the type of the resource
  */
-function getResources(server, type, ids = null) {
-  const id_param = ids ? `&_id=${ids.join(',')}` : '';
-  const url = `${server}/${type}?_include=*${id_param}`;
-  const headers = { Authorization: 'Bearer admin' }; // TODO: get an access token
-  return axios
+async function getResources(server, resourceType, db) {
+  const url = `${server}/${resourceType}?_include=*`;
+  const token = await connectToServer(server);
```

Comment:
There seems to be several calls to `connectToServer`, and it asks for another access_token each time.  Is there a way we can just use the same token or refresh the token if the URL is the same?

Reply:
That's Mike's PR #11